lib/saml_idp/version.rb

```diff
@@ -1,4 +1,4 @@
 # encoding: utf-8
 module SamlIdp
-  VERSION = '0.5.0-18f'
+  VERSION = '0.6.0-18f'.freeze
```

Comment:
is this on purpose?  this tree is based off a fork of 0.5.0 and I don't think this commit pulls in all the stuff that's in 0.6.0 of https://github.com/saml-idp/saml_idp